restconnector/RequestHandler.h

```diff
@@ -12,24 +12,38 @@
 namespace restconnector
 {
 
+/**
+ * The RequestHandler class handles incoming HTTP requests and
+ * transforms them into corresponding zeq events based on registered
+ * vocabularies.
+ */
 class RequestHandler
 {
 public:
 
-    RequestHandler();
+    /**
+     * Constructor
+     * @param publisherSchema Schema on which zeq events will be published
+     * @param subscriberSchema Schema on which zeq events will be recieved
+     */
+    RequestHandler( const std::string& publisherSchema,
+                    const std::string& subscriberSchema );
 
     ~RequestHandler();
 
-    /*<< This is the function that handles the incoming request. >>*/
+    /**
+     * This operator in invoked whenever an HTTP request is accepted by the server.
+     * It is responsible for processing the request and generating the response.
+     * @param request Incoming HTTP request
+     * @param response Response to be sent back
+     */
     void operator() ( const server::request &request, server::response &response );
 
-    void unlock();
-
-    /*<< It's necessary to define a log function, but it's ignored in
-         this example. >>*/
-    void log(...) {
-        // do nothing
-    }
+    /**
+     * Callback method used for logging internal errors.
+     * @param info String containing the error message
+     */
+    void log( server::string_type const &info );
```

Comment:
The log method is called by the underlying cppnetlib library whenever an error is detected at a socket level. I added this to the documentation.